### PR TITLE
fix: adapt dfl parser to different format versions and fix statsperform gameclock unit

### PR DESCRIPTION
**Pull request by hbiermann95**
- update dfl parser to newest version
- convert gameclock of statsperform parser from milliseconds to seconds

### CODE REVIEW

File: floodlight/io/dfl.py [hunk 1]
```diff
@@ -375,25 +375,33 @@ def read_event_data_xml(
...
-    # find start and end of half
-    kickoff_whistles = {}
-    final_whistles = {}
-    for whistle in root.findall("Event/KickoffWhistle"):
-        if whistle.get("GameSection") is not None:
-            kickoff_whistles[whistle.get("GameSection")] = iso8601.parse_date(
-                whistle.getparent().get("EventTime")
+    # find start of halves
+    start_times = {}
+    start_events = root.findall("Event/KickoffWhistle")
```
Comment (draabe) [inline floodlight/io/dfl.py:380]: maybe a comment would be helpful that this is a looking for two different encodings of whistles as the data format changed over time

**Review by draabe — COMMENTED** [review]
just two minor observations

File: floodlight/io/dfl.py [hunk 2]
```diff
@@ -375,25 +375,33 @@ def read_event_data_xml(
...
+    if not bool(start_events):  # if no KickoffWhistle is in data search for Kickoff
+        start_events = root.findall("Event/Kickoff")
+    if not bool(start_events):  # if no Kickoff is in data search for KickOff
+        start_events = root.findall("Event/KickOff")
```
[the remark this answers is not in the document]
Comment (hbiermann95) [reply floodlight/io/dfl.py:384]: KickOff (apparently in newer data) vs. Kickoff (older data)

[the remark this answers is not in the document]
Comment (draabe) [reply floodlight/io/dfl.py:385]: ah, I misread that

**Review by draabe — APPROVED** [review]

**Pull request merged**